pkg/grok/docker_compose.go

```diff
@@ -113,15 +113,19 @@ func ResolveDockerComposeDependencies(env *build.BuilderContext, projectDirector
 		} else {
 			dockerfilePath = filepath.Join(imageContext, env.Expand(service.Build.Dockerfile))
 		}
-		runtime, buildtime, err := ResolveDockerfileDependencies(dockerfilePath)
+		runtime, buildtimes, err := ResolveDockerfileDependencies(dockerfilePath)
```

Comment:
Nit: ImageDependencies use Buildtime, better be consistent

Reply:
I usually name it buildtimes when I need to iterate. I guess I am not doing it here

Reply:
Fixed